doc/using_the_march_exoskeleton/how_to_airgait.rst

```diff
@@ -21,40 +21,42 @@ The Setup
 
 Start up
 ---------
-- **Low voltage:** Make sure the high voltage if off, by pressing the emergency button.
-  Give the |march| low voltage, by enabling power.
-- **Start the exoskeleton pc:** Keep the start button pressed until the blue light is continuously on.
-- The computer is now booting. The display, mouse and keyboard should automatically connect.
-- **High voltage:** Turn the high voltage on, but releasing the emergency button.
-- **Start the software:** Type in a sourced terminal :code:`roslaunch march_launch march.launch`.
-- **Select launch settings:** Select in the launch menu the settings you need.
-
-
-Remote Airgait
+#. **Low voltage:** Make sure the high voltage is off, by pressing the emergency button.
+   Give the |march| low voltage, by enabling power.
+#. **Start the exoskeleton pc:** Keep the start button pressed until the blue light is continuously on.
+   The computer is now booting. The display, mouse and keyboard should automatically connect.
+#. **High voltage:** Turn the high voltage on, by releasing the emergency button.
```

Comment:
The low voltage net is gone, so this should be changed to having high-voltage from the start.